pulpcore/pulpcore/app/models/task.py

```diff
@@ -161,21 +194,33 @@ class TaskLock(Model):
     Fields:
 
         name (models.TextField): The name of the item that has the lock
-        timestamp (models.DateTimeField): The time the lock was acquired
+        timestamp (models.DateTimeField): The time the lock was last updated
         lock (models.TextField): The name of the lock acquired
 
     """
-    CELERY_BEAT = 'CeleryBeat'
     RESOURCE_MANAGER = 'ResourceManager'
     LOCK_STRINGS = (
-        (CELERY_BEAT, 'Celery Beat Lock'),
-        (RESOURCE_MANAGER, 'Resource Manager Lock')
+        (RESOURCE_MANAGER, 'Resource Manager Lock'),
     )
 
     name = models.TextField(db_index=True, unique=True)
-    timestamp = models.DateTimeField(auto_now_add=True)
+    timestamp = models.DateTimeField(auto_now=True)
     lock = models.TextField(unique=True, null=False, choices=LOCK_STRINGS)
 
+    def update_timestamp(self):
+        """
+        Update the timestamp field to now and save it.
+
+        Warnings:
```

Comment:
This looks like a Napolean style doc aspect but it's not. Can `Warnings:` not be used and instead move that text to be just another sentence on L#214?